nixos/modules/services/blockchain/nimbus/beacon-node.xml

```diff
@@ -0,0 +1,61 @@
+<chapter xmlns="http://docbook.org/ns/docbook"
```

Comment:
New documentation should no longer be in docbook which we are slowly removing but in markdown and for now needs a translation to docbook.

Reply:
Can you explain more about what that means? Any wiki page you can link to about docbook usage in `nixpkgs`?

Reply:
What do you mean by "translation to docbook"? What should I do?

Reply:
https://github.com/NixOS/nixpkgs/blob/master/doc/contributing/contributing-to-documentation.chapter.md

Reply:
Thanks.

Reply:
So by "translation to docbook" do you mean there's some kind of script that can convert the markdown to docbook? Or do i need to commit both the XML and the Markdown files?

Reply:
Also, the doc XML errors make zero sense to me.